brax/training/agents/ppo/train.py

```diff
@@ -267,6 +283,36 @@ def train(
   gradient_update_fn = gradients.gradient_update_fn(
       loss_fn, optimizer, pmap_axis_name=_PMAP_AXIS_NAME, has_aux=True)
 
+  def random_translate_pixels(obs: Mapping[str, jax.Array], key):
```

Comment:
nit: type annotation on key

also, does this function need to live inside `train()` or can it be moved outside?

Reply:
It might be worth having an image_augmentations.py for future augmentations (color shifts etc.)